src/pages/HomePage.js

```diff
@@ -0,0 +1,197 @@
+import "../styles/homepage.css";
+
+const HomePage = () => {
+    return(
+        <div>
+            <main>
+        <section className="section section-main">
+            <div>
+            <img className="image-section"
+                src="https://cdn.shortpixel.ai/spai/q_+ret_img+to_webp/https://www.banarasisaree.com/wp-content/uploads/2021/11/slider-home-wedding-2nw.jpg"
```

Comment:
you can also use below code for image :--
import img_name from "..../...//path" 
.
.
<img src={img_name}  />